pgjdbc/src/main/java/org/postgresql/core/Parser.java

```diff
@@ -416,122 +412,6 @@ public static int parseBlockComment(final char[] query, int offset) {
   }
 
   /**
-   * Parse string to check presence of DELETE keyword regardless of case.
-   * The initial character is assumed to have been matched.
-   * @param query char[] of the query statement
-   * @param offset position of query to start checking
-   * @return boolean indicates presence of word
-   */
-  public static boolean parseDeleteKeyword(final char[] query, int offset) {
```

Comment:
Please don't substitute the code to coerce the char. Dave indicated this technique is necessary.
https://github.com/pgjdbc/pgjdbc/pull/491#issuecomment-209350929